src/main/webapp/WEB-INF/jsp/submit/uploadFileToMirth.jsp

```diff
@@ -129,7 +129,11 @@
 		fileObjects= f.listFiles();
 	}	
 	
-    
+    Arrays.sort(fileObjects, new Comparator<File>() {
+    public int compare(File f1, File f2) {
+        return Long.compare(f2.lastModified(), f1.lastModified());
+    }
+	});
```

Comment:
Instead of adding the comparison logic here, can you update importDataHelper.getPersonalImportLogFile() method to return files sorted? That way, not just this page but any other pages that use this method will have the same consistent sorting. 
In general, it's not a good idea to use scriptlets in jsps. See "JSP Scriptlets" section here: https://www.oracle.com/technical-resources/articles/javase/code-convention.html

Reply:
Done